invokeai/backend/model_management/model_probe.py

```diff
@@ -64,7 +64,7 @@ def heuristic_probe(cls,
     @classmethod
     def probe(cls,
               model_path: Path,
-              model: Union[Dict, ModelMixin] = None,
+              model: Optional[Union[Dict, ModelMixin]],
```

Comment:
You sure that there no need in default argument?
On 58 line:
```python
return cls.probe(model_path=model,prediction_type_helper=prediction_type_helper)
```
it called without model argument and with current code should raise and exception.

Reply:
Actually I caught that earlier today and fixed it.